src/main/java/pl/piekoszek/gorskimatches/token/AccountController.java

```diff
@@ -1,18 +1,29 @@
 package pl.piekoszek.gorskimatches.token;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
+import pl.piekoszek.gorskimatches.repository.AccountRepository;
 
+@CrossOrigin
```

Comment:
why?

Reply:
Good question, I think I was prepping it from some code and thought it was needed to work.